### PR TITLE
Document verbose flag when performing an update

### PR DESCRIPTION
**Pull request by Yasumoto**
This seemed like the right spot for this- please feel free to suggest alternatives!

### CODE REVIEW

File: Documentation/Resources.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ Also, include the verbose build log. If you're using `swift build` to compile th
 
     $ swift build -v
 
+If you are finding interesting behavior when updating your package, you can get verbose information via:
+
+    $ swift package --verbose update
```
Comment (aciidb0mb3r) [inline Documentation/Resources.md]: We already have verbose explained above, so I don't think this is needed.

Comment (Yasumoto) [reply Documentation/Resources.md]: I was having a tough time finding it (via `git grep` and github search) when looking for package resolution information- would this be better as

```
add `--verbose` or `-v` after the subcommand to obtain the verbose log, such as:

    $ swift build -v
```

If not, no worries and feel free to close! 🙇

Comment (aciidb0mb3r) [reply Documentation/Resources.md]: Yeah, that makes sense! Can you update the PR?

Comment (Yasumoto) [reply Documentation/Resources.md]: Thanks! Tried to incorporate it a bit better

**Comment by aciidb0mb3r** [comment]
Thanks! I squashed and slightly changed the wordings.

**Comment by Yasumoto** [comment]
Perfect- thank you! ✌️

**Pull request merged**